content/03.methods.md

```diff
@@ -43,10 +43,13 @@ Following training, we generated a document vector for every article within bioR
 This document vector is calculated by taking the average of every token present within a given article, ignoring those that were absent from the word2vec model.
 
 #### Dimensionality Reduction of Document Embeddings
-1. Explain how tSNE works (paragraph one)
-2. Explain how PCA works  (paragraph two)
-3. Discuss how words were mapped onto PC components via cosine similarity
-4. ^ Explain cosine similarity
+We used t-Distributed Stochastic Neighbor Embedding (tSNE) [@raw:tsne_Maaten] to generate a global view of the bioRxiv preprint landscape. 
```

Comment:
I thought the plan was to use PCA for interpretability. Are you still planning to leave tSNE in as well?

Reply:
Ah. I was under the impression to include it; however, now that I think about it sticking with PCA only is the better choice.